stories/Containers/VendorsTable.stories.tsx

```diff
@@ -8,20 +15,92 @@ export default {
     component: VendorsTable,
 } as Meta;
 
+const hasHoverMode = false;
+
 const getVendorsTableProps = () => ({
+    // columns and data must be memoized
+    columns: [
+        {
+            Header: 'Client',
+            accessor: 'col_client',
+            Cell: cell => (
+                <Profile 
+                    key={cell.row.original.id} 
+                    name={cell.row.original.name} 
+                    email={cell.row.original.email} 
+                    imageUrl={cell.row.original?.imageUrl} 
+                />
+            )
+        },
+        {
+            Header: 'Tags',
+            accessor: 'col_tags',
+            Cell: cell => (
+                <TagContainer 
```

Comment:
Had questions on how to access the props' style. For instance, I need to add margin between the props and I need iconProps' style, or is there a method for doing so?

Reply:
What do you mean here, I would like this to be explained a bit clearer a call might be helpful as well.